targets/science/include/PinNames.h

```diff
@@ -28,7 +28,8 @@
  *******************************************************************************
  */
 
-#pragma once
+#ifndef MBED_PINNAMES_H
+#define MBED_PINNAMES_H
```

Comment:
pragma once is better. it does the same thing and ensures no name clashes

Reply:
done